ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands.java

```diff
@@ -1352,7 +1351,7 @@ public void run() {
       }
     }, 5000);
     long start = System.currentTimeMillis();
-    runStatementOnDriver("alter table "+ TestTxnCommands2.Table.ACIDTBL +" compact 'major' AND WAIT");
+    runStatementOnDriver("alter table "+ Table.ACIDTBL +" compact 'major' AND WAIT");
```

Comment:
nit: Maybe OCD but put spaces around the `+` signs please

Reply:
fixed